tests/ui/target-feature/tied-features.rs

```diff
@@ -23,6 +24,7 @@ pub fn main() {
 
 #[target_feature(enable = "paca")]
 //~^ ERROR must all be either enabled or disabled together
+//~| ERROR must all be either enabled or disabled together
```

Comment:
It looks like we now emit this lint twice. I don't know why this happens...

... at least it won't be observable on stable due to diagnostics deduplication...

Reply:
Ah, that error is emitted in `llfn_attrs_from_instance`. What a hack... so each codegen backend has to re-implement the same logic. :/

Anyway, that makes it an easy albeit hacky fix.